src/common/transformers/PREFIX.js

```diff
@@ -1,34 +1,10 @@
-export const prefix = (value, str) => {
-    if (!value) {
-        return str;
-    }
+import { transformerWithArg } from './transformer';
 
-    if (typeof value === 'string') {
-        return str.concat(value);
-    }
+export const prefix = (value, str) =>
+    typeof value === 'string' ? str.concat(value) : value;
```

Comment:
This prevents to use PREFIX on an array, we have to update [the documentation](https://www.gitbook.com/book/lodex/lodex-user-documentation/edit#/edit/master/Administration/Mod%C3%A8le/Transformers/PREFIX.md).

Reply:
We can use it on an array, like all others transformers :
BOOLEAN, CAPITALIZE, LOWERCASE, NUMBER, STRING, TRIM, UPPERCASE, PREFIX, SUFFIX, REPLACE

See 
https://github.com/Inist-CNRS/lodex/blob/a6b2bce09288a7ecf49b4d7ae2bb9d433adbd714/src/common/transformers/transformer.js#L4-L8

Reply:
but the documentation will sould be update, because, now, all "string" transformers will be apply on each item of the array. It's more constitent et more usefull.
If we need to add item on array, i think it's better to add a specific transfomer

Reply:
Oh! I did not notice that.
Smarter than I thought. :)